src/components/header.js

```diff
@@ -11,39 +14,130 @@ const menus = [
   { label: "Jobs", path: "/jobs" },
 ]
 
-const Header = ({ siteTitle }) => (
-  <Box
-    as="header"
-    sx={{
-      backgroundColor: "transparent",
-      position: "absolute",
-      top: 0,
-      left: 0,
-      right: 0,
-      zIndex: 999,
-    }}
-  >
-    <Container>
-      <Flex
-        color="white"
-        justifyContent="space-between"
-        alignItems="center"
-        py={2}
-      >
-        <GatsbyLink to="/">
-          <Image src={Logo} alt={siteTitle} width="12.5rem" />
-        </GatsbyLink>
-        <Box>
-          {menus.map(menu => (
-            <Link key={menu.label} variant="nav" href={menu.path}>
-              {menu.label}
-            </Link>
-          ))}
-        </Box>
-      </Flex>
-    </Container>
-  </Box>
-)
+const Header = ({ siteTitle }) => {
+  const size = useWindowSize()
+  const [isMobile, setIsMobile] = useState()
```

Comment:
isMobile is a derived state and an unnecessary rerender. (unless useWindowSize is just a reference)

`const isMobile = (size.width < 640) `